.gitattributes

```diff
@@ -5,4 +5,16 @@
 *.jpg binary
 
 #Files we do not want to distribute
-/tests export-ignore
\ No newline at end of file
+/.appveyor            export-ignore
```

Comment:
What do you think about sorting by

* kind (directories, files)
* name

?

Also note that some of the entries do not match existing files or directories.

```diff
diff --git a/.gitattributes b/.gitattributes
index dc46e75..7044b87 100644
--- a/.gitattributes
+++ b/.gitattributes
@@ -4,17 +4,21 @@
 *.png binary
 *.jpg binary

-#Files we do not want to distribute
-/.appveyor            export-ignore
-/.build               export-ignore
-/.ci                  export-ignore
-/.codecov.yml         export-ignore
+# Directories and files we do not want to distribute
+/.build/              export-ignore
+/.ci/                 export-ignore
+/.github/             export-ignore
+/devTools/            export-ignore
+/tests/               export-ignore
 /.editorconfig        export-ignore
 /.gitattributes       export-ignore
-/.github              export-ignore
 /.gitignore           export-ignore
 /.php_cs.dist         export-ignore
 /.travis.yml          export-ignore
+/appveyor.yml         export-ignore
+/box.json.dist        export-ignore
+/codecov.yml          export-ignore
 /infection.json.dist  export-ignore
+/Makefile             export-ignore
 /phpunit.xml.dist     export-ignore
-/tests                export-ignore
\ No newline at end of file
+/setup_environment.sh export-ignore
```

Reply:
I think i applied the changes.